config/upload-api.yml

```diff
@@ -332,6 +332,35 @@ paths:
 
   /v1/area/{upload_area_id}/{filename}:
 
+    post:
```

Comment:
I think people are going to get confused between POST file and PUT file.
We might want to restructure out API at some point.
This is just me editorializing for now.